### PR TITLE
fix(initialization): choose biometrics screen

### PR DESCRIPTION
**Pull request by banklesss**
## Description / Change(s) / Related issue(s)

choose biometrics screen adatation to the new flow

##### Ticket

YOMO-1485

> [!NOTE]
> Please create the ticket if missing it.

### CODE REVIEW

File: apps/wallet-mobile/src/features/Initialization/ChooseBiometricLogin/ChooseBiometricLoginScreen.tsx [hunk 1]
```diff
@@ -9,40 +8,27 @@ import {useQuery, UseQueryOptions} from 'react-query'
...
   const strings = useStrings()
-  const storage = useAsyncStorage()
-  const {walletImplementationIdChanged} = useSetupWallet()
-  const {resetToWalletSetupInit} = useWalletNavigation()
 
-  const navigate = () => {
-    walletImplementationIdChanged(HASKELL_SHELLEY.WALLET_IMPLEMENTATION_ID)
-    resetToWalletSetupInit()
-  }
+  const {setScreenShown, isLoading: isSetScreenShownLoading} = useSetScreenShown()
```
Comment (michalszorad) [inline apps/wallet-mobile/src/features/Initialization/ChooseBiometricLogin/ChooseBiometricLoginScreen.tsx]:
```suggestion
  const {setScreenShown, isLoading: isScreenShownLoading} = useSetScreenShown()
```

**Review by michalszorad — APPROVED** [review]

**Review by jorbuedo — APPROVED** [review]

File: apps/wallet-mobile/src/features/Portfolio/useCases/PortfolioTokenDetails/PortfolioTokenDetailsScreen.tsx [hunk 2]
```diff
@@ -41,6 +40,10 @@ export const PortfolioTokenDetailsScreen = () => {
     active: activeTab === 'transactions',
   })
 
+  React.useEffect(() => {
+    track.portfolioTokenDetails({token_details_tab: tabs[activeTab]})
+  }, [activeTab, track])
+
```
Comment (banklesss) [inline apps/wallet-mobile/src/features/Portfolio/useCases/PortfolioTokenDetails/PortfolioTokenDetailsScreen.tsx:46]: removed after merging a PR before

File: apps/wallet-mobile/src/features/Portfolio/useCases/PortfolioTokenDetails/PortfolioTokenDetailsScreen.tsx [hunk 3]
```diff
@@ -28,7 +28,6 @@ export const PortfolioTokenDetailsScreen = () => {
   const {styles} = useStyles()
   const strings = useStrings()
   const {activeTab, setActiveTab} = usePortfolioTokenDetailContext()
-  const scrollY = React.useRef(new Animated.Value(0)).current
```
Comment (banklesss) [inline apps/wallet-mobile/src/features/Portfolio/useCases/PortfolioTokenDetails/PortfolioTokenDetailsScreen.tsx]: unused

File: apps/wallet-mobile/src/features/Portfolio/useCases/PortfolioTokenDetails/PortfolioTokenDetailsScreen.tsx (hunk 2 above)
Comment (stackchain) [reply apps/wallet-mobile/src/features/Portfolio/useCases/PortfolioTokenDetails/PortfolioTokenDetailsScreen.tsx:46]: @banklesss just dropped after merging fixes from VAR, it was not in use, can you merge develop again?

Comment (banklesss) [reply apps/wallet-mobile/src/features/Portfolio/useCases/PortfolioTokenDetails/PortfolioTokenDetailsScreen.tsx:46]: done

**Pull request merged**